### PR TITLE
Res 559 export installed packages in weekly built base images

### PR DESCRIPTION
**Pull request by nghiant2710**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -17,6 +17,15 @@ for suite in $SUITES; do
 	
 	./mkimage.sh -t $REPO:$suite --dir=$dir debootstrap --variant=minbase --keyring=/root/.gnupg/pubring.gpg --arch=armhf --include=sudo $suite $MIRROR
 	rm -rf $dir
+	
+	docker run --rm $REPO:$suite bash -c 'dpkg-query -l' > $suite
+	cp $suite $suite_$date
```
Comment (petrosagg) [inline build.sh]: Why do you copy this file? I don't see it being used later

Comment (nghiant2710) [inline build.sh]: @petrosagg $suite is the latest file and $suite_$date keeps tracking the history after each build

File: Dockerfile [hunk 2]
```diff
@@ -6,7 +6,11 @@ RUN apt-get -q update \
 	&& apt-get -qy install \
 		curl \
 		docker.io \
-		debootstrap
+		debootstrap \
+		python \
+		python-pip
```
Comment (Page-) [inline Dockerfile]: `&& rm -rf /var/lib/apt/lists/*`

Comment (nghiant2710) [inline Dockerfile]: @Page- Updated!

File: build.sh (hunk 1 above)
Comment (petrosagg) [inline build.sh]: No need to do that, just run this:
```bash
aws s3 cp $suite s3://$BUCKET_NAME/image_info/rpi-raspbian/$suite/
aws s3 cp $suite s3://$BUCKET_NAME/image_info/rpi-raspbian/$suite/$suite_date
rm $suite
```

Comment (nghiant2710) [inline build.sh]: Yup. Updated!

**Pull request merged**